library/src/lib/tabs/tab-list.component.html

```diff
@@ -1,17 +1,15 @@
 <ul class="fd-tabs"
     role="tablist">
-    <li class="fd-tabs__item"
-        *ngFor="let tab of tabs.toArray(); let i = index;">
+    <li fd-tab-item
+        *ngFor="let tab of panelTabs.toArray(); let i = index;">
         <a #tabLink
-           role="tab"
-           class="fd-tabs__link"
+           fd-tab-link
+           [disabled]="tab.disabled"
+           [active]="tab.expanded"
            [attr.tabindex]="tab.disabled ? -1 : 0"
            [attr.aria-controls]="tab.id"
-           [attr.aria-disabled]="tab.disabled"
```

Comment:
The aria stuff serves as more than just a visual thing here, it's for accessibility. Though I haven't really tested this component with a screenreader so not sure how much of an impact they have